android/src/main/java/com/reactlibrary/IndySdkModule.java

```diff
@@ -60,21 +62,15 @@
     private static final String TAG = "IndySdk";
     private final ReactApplicationContext reactContext;
 
-    private Map<Integer, Wallet> walletMap;
-    private Map<Integer, Pool> poolMap;
-    private Map<Integer, WalletSearch> searchMap;
-    private Map<Integer, CredentialsSearchForProofReq> credentialSearchMap;
-
-    // Java wrapper does not expose credentialSearchHandle
-    private int credentialSearchIterator = 0;
```

Comment:
Is there a specific reason you removed these? Seems unrelated to this change

Reply:
oops! No! Good catch! Let me fix it! :)